plugins/generator-1.14.4/forge-1.14.4/procedures/entity_getsaturation.java.ftl

```diff
@@ -0,0 +1 @@
+((${input$entity} instanceof PlayerEntity)?((PlayerEntity)${input$entity}).getFoodStats().getSaturationLevel():0)
```

Comment:
Please rename this and all corresponding files (the 1.15.2 generator and the block definitions) file to "entity_get_saturation".

Reply:
Done.